src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/use-trips-query.ts

```diff
@@ -364,3 +367,16 @@ function useJourneySearchModes(
       : defaultValue,
   };
 }
+
+function transportModeToEnum(
+  modes: TravelSearchTransportModes[],
+): Types.TransportModes[] {
+  return modes.map((internal) => {
+    return {
+      transportMode: internal.transportMode as Types.TransportMode,
+      transportSubModes: internal.transportSubModes?.map(
+        (submode) => submode as Types.TransportSubmode,
+      ),
```

Comment:
Possible to use helper `enumFromString` (which returns `undefined` if it doesn't exist) here, to have extra safety if at some point there is a mismatch between config-specs and the journeyplanner-types?

Reply:
Hmm. Is there a mismatch now? 🤔 I thought it could be undefined. But I'll control check

Reply:
Ah, now I get what you meant... Fixed